net/net.sh

```diff
@@ -232,11 +232,11 @@ build() {
   echo "Build took $SECONDS seconds"
 }
 
-SOLANA_HOME="\$HOME/solana"
-CARGO_BIN="\$HOME/.cargo/bin"
-
 startCommon() {
   declare ipAddress=$1
+  remoteHome="$(ssh "${sshOptions[@]}" "$ipAddress" "echo \$HOME")"
+  SOLANA_HOME="${remoteHome}/solana"
+  CARGO_BIN="${remoteHome}/.cargo/bin"
```

Comment:
*suggestion*

Maybe also `s/SOLANA_HOME/REMOTE_SOLANA_HOME/g`, and `s/CARGO_BIG/REMOTE_CARGO_BIN/g`?
To make it clear which side these variables are for.
There is `SOLANA_ROOT` that holds the local path, and it might not be clear which is which, when they are next to each other.

Reply:
Added `REMOTE_` to the variable names.